pubspec.lock

```diff
@@ -958,10 +958,10 @@ packages:
     dependency: transitive
     description:
       name: web
-      sha256: afe077240a270dcfd2aafe77602b4113645af95d0ad31128cc02bce5ac5d5152
+      sha256: dc8ccd225a2005c1be616fe02951e2e342092edf968cf0844220383757ef8f10
       url: "https://pub.dev"
     source: hosted
-    version: "0.3.0"
+    version: "0.1.4-beta"

```

Comment:
Je to spravne ze se ten soubor zmenil? 🤔

Reply:
Je zvláštní že se to downgradovalo. Spustil jsem ted `flutter pub upgrade`  to by mělo zajistit aktualizaci závislostí. Asi by se měl snažit každý vývojář udržovat závislosti aktuální aby k tomu nedocházelo?
Dám si poznámku a probereme na COP.